src/ensembl/src/shared/components/tabs/Tabs.test.tsx

```diff
@@ -58,82 +59,97 @@ const defaultProps = {
   classNames: tabClassNames
 };
 
-const renderTabs = (props: TabsProps) => <Tabs {...props}></Tabs>;
+const renderTabs = (props: TabsProps) => render(<Tabs {...props} />);
```

Comment:
I guess we don't need to `props` param here as we are only using `defaultProps` in all the test cases.

Reply:
✅